LocalSettings.php

```diff
@@ -2860,6 +2860,23 @@
 				'edittemplateprotected' => true,
 			],
 		],
+		'+nycsubwaywiki' => [
+			'steward' => [
+				'managewiki-restricted' => true,
+				'interwiki' => true,
+				'siteadmin' => true,
+				'bigdelete' => true,
+				'renameuser' => true,
```

Comment:
Renames are done at a global scale, local renames aren't ever practiced and no group has the right in their toolset so this can be removed.